### DISCUSSION TITLE
Pengurusan Kewangan Masjid

### DISCUSSION THREAD

**Discussion by rohaizanr** [Dev4w4n/e-masjid.my#125] [Ideas]
Report apa yang perlu ?
Data apa perlu simpan ?
UI design ?

**Comment by imie** [comment]
Boleh design summary duit masuk dan duit keluar untuk page depan. 

Maybe boleh tambah duit tunai dalam tangan dan duit dalam bank ada berapa baki.

Duit Keluar Masuk tu boleh buat total dan link ke page itemized. Macam bila click ke Sewaan, pergi ke satu page yang memaparkan list pembayaran sewaan barang untuk latest tahun dan ada tab untuk tahun2 sebelum nya.

![image](https://github.com/Dev4w4n/e-masjid.my/assets/2321338/4aa176c5-a108-4e21-9dcc-c35952e955df)


Dan boleh tambah link 'claimable expenses' untuk AJK claim semula pembelian alatan surau. Boleh upload resit untuk bukti pembelian.

![image](https://github.com/Dev4w4n/e-masjid.my/assets/2321338/68b5713c-5170-47b4-8138-eddab1af0c45)

**Comment by rohaizanr** [comment]
terima kasih @imie

**Comment by rohaizanr** [comment]
Module AJK kena siap dulu @jebatco

**Comment by DeenSkuad** [comment]
<img width="1520" alt="e-masjid-Kewangan-mockup" src="https://github.com/Dev4w4n/e-masjid.my/assets/157102745/ee617231-38b9-4209-9451-128b7c5c0b7f">

nanti boleh share in detail apa yg perlu ada dalam page kewangan.

**Comment by dinayub** [comment]
Statistik Kewangan

<img width="1520" alt="Kewangan" src="https://github.com/Dev4w4n/e-masjid.my/assets/164418432/49f17172-22e7-4170-a7bf-1de44ef1215c">

**Comment by rohaizanr** [comment]
Untuk rujukan module kewangan

[Garis_Panduan_Penyelarasan_Pengurusan_Kewangan_Masjid_dan_Surau_Seluruh_Malaysia.pdf](https://github.com/Dev4w4n/e-masjid.my/files/14727178/Garis_Panduan_Penyelarasan_Pengurusan_Kewangan_Masjid_dan_Surau_Seluruh_Malaysia.pdf)
